flexmeasures/data/schemas/scheduling/storage.py

```diff
@@ -54,7 +54,8 @@ class StorageFlexModelSchema(Schema):
     You can use StorageScheduler.deserialize_flex_config to get that filled in.
     """
 
-    soc_at_start = fields.Float(required=True, data_key="soc-at-start")
+    soc_at_start = fields.Float(required=False, data_key="soc-at-start")
+    soc = QuantityOrSensor("MWh", data_key="soc", required=False)
```

Comment:
Correct. And for the scope of this PR, we'll simply ignore any SoC that is passed as a quantity.